.gitignore

```diff
@@ -72,7 +72,5 @@ target/
 venv/
 ENV/
 
-# Elastic Beanstalk Files
-.elasticbeanstalk/*
-!.elasticbeanstalk/*.cfg.yml
-!.elasticbeanstalk/*.global.yml
+# Fabric Files
+fabfile/
```

Comment:
you added fabfile folder. But there are no files in that folder. This folder doesn't even exist in the PR. 
Where are the fabfile settings that got configured successfully.